### PR TITLE
Use langtool to add missing MemStick sections to a zillion ini files

### PR DESCRIPTION
**Pull request by hrydgard**
Langtool: #14894 

Not sure if I should hold off on this until we really have finalized the dialog..

Just realized I can also use the tool to remove outdated strings with some additions... but enough for today. making this a draft for now.

### CODE REVIEW

File: assets/lang/ar_AE.ini [hunk 1]
```diff
@@ -577,6 +578,7 @@ Browse = ‎تصفح...
 Buy PPSSPP Gold = Buy PPSSPP Gold
 Choose folder = Choose folder
 Credits = ‎الحقوق
+PPSSPP Homebrew Store = PPSSPP Homebrew Store
```
Comment (unknownbrackets) [inline assets/lang/ar_AE.ini:581]: Hmm... duplicate?

-[Unknown]

Comment (hrydgard) [reply assets/lang/ar_AE.ini:581]: Huh, that's interesting, no idea what it did there.

Anyway I'm gonna tweak the tools a bit more and re-run, but not today.

**Comment by hrydgard** [comment]
Oh yeah, next step for the tool will be to scan the source code for translation strings. Can just look for `->T("` ...

File: assets/lang/ar_AE.ini (hunk 1 above)
Comment (hrydgard) [reply assets/lang/ar_AE.ini:581]: There's a zero-width space at the start of this line. This tool seems to be good at indirectly finding that kind of weirdness :)

**Comment by hrydgard** [comment]
Cleaned up some errors in the ini files, and added a couple of exceptions to langtool. Output is now very clean.

**Comment by ResurrectionRedmi9** [comment]
Added to v1.12.0 milestone.

**Pull request merged**